Extension/src/main.ts

```diff
@@ -72,9 +72,20 @@ export async function activate(context: vscode.ExtensionContext): Promise<CppToo
 
     await processRuntimeDependencies();
 
-    // check if the correct offline/insiders vsix is installed on the correct platform
     const installedPlatform: string | undefined = util.getInstalledBinaryPlatform();
-    if (!installedPlatform || (process.platform !== installedPlatform)) {
+
+    // Check the main binaries files to declare if the extension has been installed successfully.
+    if (!installedPlatform || !(await util.checkInstallBinariesExist())) {
+        errMsg = localize("extension.installation.failed", "The C/C++ extension failed to install successfully. You will need to repair or reinstall the extension for C/C++ language features to function properly.");
+        const reload: string = localize("remove.extension", "Attempt to Repair");
+        vscode.window.showInformationMessage(errMsg, reload).then(async (value?: string) => {
```

Comment:
Why is this an information message and not an error message?

Reply:
Do you think the error message will be better?

When the **VSIX** belongs to another platform => **error** message (go to the download page)
When some **Json** files are missing                  => **warning** message (go to the download page)
When some **Binaries** are missing                    => **warning** message (attempt to repair)

Reply:
I changed them all to error messages